support-frontend/test/selenium/util/PostDeployTestUser.scala

```diff
@@ -24,3 +28,31 @@ class PostDeployTestUser(driverConfig: DriverConfig) extends TestUser {
   val username = testUsers.generate()
   addTestUserCookies(username)
 }
+
+class PostDeployTestUserSubs(driverConfig: DriverConfig) extends TestUser {
+
+  private val client = new OkHttpClient()
```

Comment:
An OkHttpClient is expensive to create so you should create it as a val on an object and reuse it for all tests

Reply:
I moved the OkHttpClient into an object, but also the request itself, meaning that all tests would now use the same test user cookies. Not sure if that makes more sense, or if each test should re-request a fresh set of cookies (using the same singleton OkHttpClient)

Reply:
I think it might be better to get a new set for each test because for Digital Sub if the user already has a subscription and tries to checkout again then they are directed to a different page. This would mean that if we had several Digisub tests (eg. for different currencies) the second one wouldn't be able to access the checkout, it could be quite confusing for whoever was trying to write the test.

Reply:
Got it, definitely makes sense. Fixed.